README.md

```diff
@@ -73,6 +73,10 @@ If you ever need help, feel free to ask around on our [Discord server](https://d
    and the backend server at http://localhost:8080 (if started).
    As you make changes to the React application in `src`, those changes will be automatically reflected on the website locally.
 
+#### Running Tests
+1. go into `apps/antalmanac`
+2. `pnpm run test`
```

Comment:
`pnpm test` also works, since pnpm will assumes "run" if there's no valid matching command.

Reply:
We can work this out in https://github.com/icssc/AntAlmanac/pull/682